### PR TITLE
Addisie mddl 003

### PR DESCRIPTION
**Pull request by Addisie**

### CODE REVIEW

**Comment by thea-m** [comment]
I suppose it corresponds to this one? https://searcharchives.bl.uk/primo_library/libweb/action/display.do?tabs=detailsTab&ct=display&doc=IAMS040-002501022&displayMode=full&vid=IAMS_VU2
If so, the ID EAP336/2/5
 can be added as an alternative identifier

File: new/MDDL003.xml [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i15">
+                            <locus from="90rb" to="90va"/> 
+                            <title type="complete" ref="LIT3320Letter"/>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i16">
+                            <locus from="90va"/> 
+                            <title type="complete" ref="LIT3331Cyril"/>
```
Comment (eu-genia) [inline new/MDDL003.xml:121]: does it really have the same text twice (i11 and i16)?

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Comment by eu-genia** [comment]
> I suppose it corresponds to this one? https://searcharchives.bl.uk/primo_library/libweb/action/display.do?tabs=detailsTab&ct=display&doc=IAMS040-002501022&displayMode=full&vid=IAMS_VU2 If so, the ID EAP336/2/5 can be added as an alternative identifier

Yes it is the same one; I will be changing shelfmarks and xml:id (the current shelfmarks and IDs will be removed as confusing) and adding facs gradually (I have already changed the institution for all)

**Comment by MershaMengistie** [comment]
Thank you, Zhenia. But I now strongly suggest (and as I noted to you yesterday), Addisie should stop working on the folders I gave her for exercise purposes. It is - at least - good that you corresponded the ones she cataloged with what are available in the folder I submitted to you. Let us work on these new and updated folders.

File: new/MDDL003.xml
Comment (MershaMengistie) [inline new/MDDL003.xml:1]: Excellent! It is.

File: new/MDDL003.xml (hunk 1 above)
Comment (Addisie) [reply new/MDDL003.xml]: I corrected it.  it was misleading by EMIP00641.

**Pull request merged**